helper/helper_functions.py

```diff
@@ -38,3 +40,13 @@ def get_db_session(db_url=DB_URL_DEFAULT):
     session = session_maker()
 
     return session
+
+
+def get_timing():
+    string_json = importlib_resources.files('virtual_coach_db').joinpath(
+        'resources/resources_timing.json').read_text()
+
+    timing = json.loads(string_json)
```

Comment:
The one thing I would expect to see here is some sort of validation of the json file to make sure it has the right structure, or that values are within accepted limits (or that phases correspond to allowed values etc). I'm not sure how thoroughly you would want to validate it but at least some might help catch bugs later

Reply:
For example, the `jsonschema` python library looks relatively easy to make and apply a schema to validate the json structure. For example, you can write a schema that looks like e.g.
```
{
  "type": "object",
  "properties": {
    "first_name": { "type": "string" },
    "last_name": { "type": "string" },
    "birthday": { "type": "string", "format": "date" },
    "address": {
      "type": "object",
      "properties": {
        "street_address": { "type": "string" },
        "city": { "type": "string" },
        "state": { "type": "string" },
        "country": { "type" : "string" }
      }
    }
  }
}
```
(an example from their docs)

and then apply that to the resources json

Reply:
Haven't used it myself though so... beware... :stuck_out_tongue:

Reply:
Oh yes, this is a great idea!